runelite-client/src/main/java/net/runelite/client/callback/Hooks.java

```diff
@@ -80,7 +81,9 @@
 
 	private static final Injector injector = RuneLite.getInjector();
 	private static final Client client = injector.getInstance(Client.class);
-	public static final EventBus eventBus = injector.getInstance(EventBus.class);
+	public static final EventBus eventBus = injector.getInstance(EventBus.class); // symbol must match mixin Hook
```

Comment:
Is this comment about why something needs to be public, or the type + name? 
I'm puzzled as to why these are public.

Reply:
It must be public because it is accessed directly from code injected into the client, which is in a different package. But the type and name must be equivalent as it is compiled against the one in the mixing and actually runs against this one.